### DISCUSSION TITLE
Tour step arrow not showing up in v10

### DISCUSSION THREAD

**Discussion by rowansiwakoti** [hakimio/ngx-ui-tour#193] [Q&A] [answered]
I am using 10.7 version with Angular 15. I cannot see the **Arrow Icon** on the dialog in any of the product tour steps. Is it expected with this version?

**The following snapshot is from official demo.**
![image](https://github.com/user-attachments/assets/bf03020a-b1e1-486e-99ca-bb0608f8b417)

**The following snapshot from my application.**
![image](https://github.com/user-attachments/assets/5a5a7286-1b20-45af-b842-85a01c1dd329)

**Comment by hakimio** [comment, marked as answer]
The feature was introduced in [v12](https://github.com/hakimio/ngx-ui-tour/releases/tag/v12.0.0)

You could copy the CSS to your app if you need it in older version:
https://github.com/hakimio/ngx-ui-tour/issues/186#issuecomment-2274242167

EDIT: you might have to adjust some CSS class names for the non-MDC based Material components.

**Reply by rowansiwakoti** [reply]
Thank you so much. It worked :)

**Reply by rowansiwakoti** [reply]
![image](https://github.com/user-attachments/assets/72268fe1-5952-42c8-948a-0aed1fcaaf0b)
Does this steps (count) also introduced in newer version? If i have to implement this in v10 then how to do it? Thank you in advance.

**Reply by hakimio** [reply]
It was also introduced in `v12`. You can add it to your tour by providing [custom step template](https://hakimio.github.io/ngx-ui-tour/md-menu/Misc#custom-template):
https://github.com/hakimio/ngx-ui-tour/blob/8d7aee2444103234150011297014184c8dea88b4/projects/ngx-ui-tour-md-menu/src/lib/tour-step-template/tour-step-template.component.html#L49

**Reply by rowansiwakoti** [reply]
Thanks a lot! :)
